### ISSUE TITLE
Fillers in bank disappears (bug)

### ISSUE THREAD

**Issue by permedd** [runelite/runelite#2555]
Its kinda hard to explain but when you left click and drag super fast all fillers gets removed, here are some gyazos to show you what i mean.

https://i.gyazo.com/ae4f4fca962f7b6a0d4fd39de51f6f43.mp4

https://i.gyazo.com/505d5872c7f9b50afa2ae2ea65b33058.mp4

**Comment by Reasel** [comment]
I am sorry to inform you, but that is not a bug. The left click option is "Clear-All. If you don't clearly drag them around it will simply click on the filler and clear all other fillers.

**Comment by permedd** [comment]
yeah i edited it xd and made it a suggestion not a bug but thanks though

**Comment by Reasel** [comment]
Do you want the default option to be cancel as well?

**Comment by raiyni** [comment]
Is this still an issue? I can't reproduce it.

**Comment by geheur** [comment]
This is a likely a jagex bug. It happens when your left mouse down is on one item, and the mouse up is on a different item, but happens before the anti-drag timer expires (so increase your anti-drag to make it easier to reproduce). What happens depends on which item you mouse downed on and which one you mouse up on. I can't replicate removing ALL fillers but I can replicate removing 1 filler by doing this. It also happens with placeholders.
I have a workaround fix for this in bank tag layouts fwiw (https://github.com/geheur/bank-tag-custom-layouts/commit/60e4566784430d7dc5ad7622b31b92bdbf4930f0) but only for placeholders..
